components/brave_wallet_ui/utils/string-utils.ts

```diff
@@ -6,7 +6,19 @@ export const toProperCase = (value: string) =>
     (txt) => txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase())
 
 export const isRemoteImageURL = (url?: string) =>
-  url?.startsWith('http://') || url?.startsWith('https://') || url?.startsWith('data:image/')
+  url?.startsWith('http://') || url?.startsWith('https://') || url?.startsWith('data:image/') || url?.startsWith('ipfs://')
 
 export const isValidIconExtension = (url?: string) =>
-  url?.endsWith('.jpg') || url?.endsWith('.jpeg') || url?.endsWith('.png') || url?.endsWith('.svg')
+  url?.endsWith('.jpg') || url?.endsWith('.jpeg') || url?.endsWith('.png') || url?.endsWith('.svg') || url?.endsWith('.gif')
```

Comment:
is this being used to check NFT urls? can we ban svg for those?

Reply:
@diracdeltas Yes, this is being used for NFT images too. Thought: Since we are using `chrome-untrusted` tab which does not allow inline javascript execution, an SVG with executable code will be blocked too.